### PR TITLE
Swedish

### PR DESCRIPTION
**Pull request by lindhe**
Most of it is correct, some minor details are a bit awkward but that's
mostly because not all technical terms ("web terms") has a proper
Swedish translation.

### CODE REVIEW

**Comment by akhodakivskiy** [comment]
Please add a record in `extension/install.rdf`

**Comment by lindhe** [comment]
I feel a bit confused on what file you mean. Do you mean `extension/install.rdf.tmpl`? That seems to be only a template. Or do you mean `extension/install.rdf` in the master branch?

**Comment by akhodakivskiy** [comment]
@lindhe - ah, nevermind about this. I missed this templating change.

@lydell - Simon, could you please describe how to transform develop branch into master now? Is this pull request all that is necessary to add new translation now?

**Comment by lydell** [comment]
You'll have to wait til tonight.

**Comment by lydell** [comment]
You could add yourself to the PEOPLE.md file, though.

**Comment by lindhe** [comment]
Cool, I feel important.

**Comment by lydell** [comment]
> Is this pull request all that is necessary to add new translation now?

Yes.

> could you please describe how to transform develop branch into master now?

You’re right, we really need to get a new version out. master is way behind develop. Actually, develop is quite stable the way it is (I’ve used it for a long time). So I’ll make a few adjustments, set my grand plans for VimFx aside for the moment, and then we’ll merge develop into master. I’ll work on this for the rest of the day.

@lindhe it’d be nice if you could squash the commits into one.

File: extension/locale/sv-SE/options.dtd [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<!ENTITY pref.hint_chars.title      "Etikettskaraktärer">
```
Comment (lydell) [inline extension/locale/sv-SE/options.dtd]: Etikettstecken?

File: extension/locale/sv-SE/options.dtd [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+<!ENTITY pref.prev_patterns.title   "Mönster för föregående sida">
+<!ENTITY pref.prev_patterns.desc    "Kommaseparerad lista av mönster som matchar länkar till föregående sida">
+
+<!ENTITY pref.next_patterns.title   "Mönster för nästa sida">
+<!ENTITY pref.next_patterns.desc    "Kommaseparerad lista av mönster som matchar länkar till nästa sida">
+
+<!ENTITY pref.scroll_step_lines.title "Scrollsteg">
+
+<!ENTITY pref.hints_bloom_on.title "Smart etikettstilldelning">
+<!ENTITY pref.hints_bloom_on.desc  "Tilldela kortare etiketter till de länkar som användaren besöker oftast">
```
Comment (lydell) [inline extension/locale/sv-SE/options.dtd]: Tilldela kortare etiketter till länkar som besöks oftast?

File: extension/locale/sv-SE/options.dtd [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+
+<!ENTITY pref.prevent_autofocus.title "Hindra autofokus på för inmatningsfält">
```
Comment (lydell) [inline extension/locale/sv-SE/options.dtd]: Remove either “på” or “för”.

File: extension/locale/sv-SE/vimfx.properties [hunk 4]
```diff
@@ -0,0 +1,85 @@
+button_tooltip_enabled=VimFx är aktiverat. Klicka för att avaktivera (Shift-Alt-v)
+button_tooltip_disabled=VimFx är avaktiverat. Klicka för att aktivera (Shift-Alt-v)
+button_tooltip_blacklisted=VimFx är svartlistad på denna sidan
```
Comment (lydell) [inline extension/locale/sv-SE/vimfx.properties]: denna sida

File: extension/locale/sv-SE/vimfx.properties [hunk 5]
```diff
@@ -0,0 +1,85 @@
...
+button_tooltip_insertMode=VimFx är i inmatningsläge på denna sidan: Alla kommandon kommer ignoreras. Klicka för att gå ur (Esc [som standard])
+item_preferences=Inställningar
+item_blacklist_button_tooltip=Svartlista
+item_blacklist_button_inverse_tooltip=Ta bort denna regeln
```
Comment (lydell) [inline extension/locale/sv-SE/vimfx.properties]: denna regel

File: extension/locale/sv-SE/vimfx.properties [hunk 6]
```diff
@@ -0,0 +1,85 @@
...
+help_command_marker_focus=Fokusera element
+help_command_yank=Kopiera länk till den nuvarande sidan till urklipp
+help_command_reload=Ladda om den nuvarande sidan
+help_command_reload_force=Ladda om den nuvarande sidan och alla resurser (js, css, img)
+help_command_reload_all=Ladda om sidor i alla flikar
+help_command_reload_all_force=Ladda om sidor i alla flikar, inklusive alla resurser (js, css, img)
+help_command_stop=Stoppa inladdning av den nuvarande sidan
+help_command_stop_all=Stoppa inladdning av sidor i alla flikar
+
+help_section_nav=Navigera sidan
```
Comment (lydell) [inline extension/locale/sv-SE/vimfx.properties]: Navigation?

File: extension/locale/sv-SE/vimfx.properties [hunk 7]
```diff
@@ -0,0 +1,85 @@
...
+help_command_follow_multiple=Öppna flera länkar på den nuvarande sidan
+help_command_follow_previous=Gå till föregående sida
+help_command_follow_next=Gå till nästa sida
+help_command_go_up_path=Gå upp en nivå i URL hierarkin
+help_command_go_to_root=Gå till roten av URL hierarkin
+help_command_back=Gå tillbaka i historiken
+help_command_forward=Gå framåt i historiken
+
+help_section_misc=Övrigt
+help_command_help=Visa denna rutan
```
Comment (lydell) [inline extension/locale/sv-SE/vimfx.properties]: denna ruta

File: extension/locale/sv-SE/vimfx.properties [hunk 8]
```diff
@@ -0,0 +1,85 @@
...
+help_command_find_hl=Gå till sökläge och markera alla sökresultat
+help_command_find_next=Gå till nästa hittade sökresultat
+help_command_find_prev=Gå till föregående hittade sökresultat
+help_command_insert_mode=Gå till inmatningsläge: Ignorera alla kommandon
+help_command_dev=Öppna utvecklarverktygsfältet
+
+help_title=Tangentbordskommandon
+help_version=Version
+
+help_overlapping_hints=Kan Du inte se en länketikett då den täcks över av en annan? Testa att klicka Mellanslag eller Shift-Mellanslag!
```
Comment (lydell) [inline extension/locale/sv-SE/vimfx.properties]: klicka → trycka på, Shift → Skift

File: extension/locale/sv-SE/vimfx.properties [hunk 9]
```diff
@@ -0,0 +1,85 @@
...
+help_feedback=Lämna feed-back!
+help_enjoying=Gillar du VimFx?
+help_found_bug=Hittat en bug?
+help_report_bug=Rapportera det här!
+help_remove_shortcut=Ta bort denna genvägen
```
Comment (lydell) [inline extension/locale/sv-SE/vimfx.properties]: denna genväg

File: extension/locale/sv-SE/vimfx.properties [hunk 10]
```diff
@@ -0,0 +1,85 @@
...
+help_remove_shortcut_title=Bekräftelse av genvägsborttagning
+help_remove_shortcut_text=Vänligen bekräfta att du vill ta bort denna genvägen
```
Comment (lydell) [inline extension/locale/sv-SE/vimfx.properties]: denna genväg

File: extension/locale/sv-SE/vimfx.properties [hunk 11]
```diff
@@ -0,0 +1,85 @@
...
+help_add_shortcut=Klicka för att lägga till en ny genväg för detta kommandot
```
Comment (lydell) [inline extension/locale/sv-SE/vimfx.properties]: detta kommando

File: extension/locale/sv-SE/vimfx.properties [hunk 12]
```diff
@@ -0,0 +1,85 @@
...
+help_add_shortcut_title=Ange ny genväg
+help_add_shortcut_text=Vänligen ange ny genväg för detta kommando. Se till att separera tangenter i flertangentskommandon med kommatecken. Mellanslag tillåts ej. Exempel på dålig inmatning 'a c-F', exempel på bra inmatning: 'a,c-F'
+help_add_shortcut_text_overwrite=Genvägen '%1S' kolliderar med följande kommando. Vill du skriva över dem?
```
Comment (lydell) [inline extension/locale/sv-SE/vimfx.properties]: kommando → kommandon

**Comment by lindhe** [comment]
> it’d be nice if you could squash the commits into one
Is that even possible after a push to GitHub?

File: extension/locale/sv-SE/options.dtd (hunk 1 above)
Comment (lindhe) [inline extension/locale/sv-SE/options.dtd]: That's much better. Thanks.

File: extension/locale/sv-SE/vimfx.properties (hunk 6 above)
Comment (lindhe) [inline extension/locale/sv-SE/vimfx.properties]: Well, I think "Navigera sidan" is clearer. "Navigation" sounds like using a GPS.
BTW, what is the best practice - writing in English or Swedish here? It does help others to understand, is there any point in it when they can not validate the discussion?

File: extension/locale/sv-SE/vimfx.properties (hunk 8 above)
Comment (lindhe) [inline extension/locale/sv-SE/vimfx.properties]: I think "klicka" feel more natural.

I changed Shift → Skift at lines 1 and 2 as well.

**Comment by lydell** [comment]
Yes, `git push -f`.

File: extension/locale/sv-SE/vimfx.properties (hunk 8 above)
Comment (lydell) [inline extension/locale/sv-SE/vimfx.properties]: In my opinion you click with the mouse and press keys; man klickar med musen och trycker på tangenter.

File: extension/locale/sv-SE/vimfx.properties (hunk 6 above)
Comment (lydell) [inline extension/locale/sv-SE/vimfx.properties]: Alright, let’s go with “Navigera sidan”.

Yes it feels a bit silly to speak English with each other about Swedish, but that’s the way things are done here.

File: extension/locale/sv-SE/vimfx.properties (hunk 8 above)
Comment (lindhe) [inline extension/locale/sv-SE/vimfx.properties]: Sure, I don't really have anything against it and maybe someone else think like you do. I'll go with that.

**Comment by lydell** [comment]
> I’ll work on this for the rest of the day.

I got pretty far, too. See #249 for example.

**Comment by lydell** [comment]
I’ve pushed lots of new stuff today, which unfortunately makes this pull request somewhat out of date. Would you mind updating it?

**Comment by lindhe** [comment]
Updating it = syncing my fork?

**Comment by lydell** [comment]
Yeah, rebase on latest develop.

**Comment by lindhe** [comment]
Sure thing.

**Comment by lindhe** [comment]
Damn, you really had made a lot of changes! :P I was so confused when I didn't find your *last* most recent commit - it turned out I had to scroll down to even see it!

**Comment by lydell** [comment]
I'm sorry that I made so many changes after your PR. I should haveerged this first. Have you statted working on rebasing? Otherwise I'll do it for you this afternoon.

**Comment by lindhe** [comment]
Oh, you made more commits. One moment.

**Pull request merged**